packages/react-scripts/template/README.md

```diff
@@ -163,7 +167,7 @@ Runs the app in the development mode.<br>
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
 The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
+You will also see any lint warnings in the console.
```

Comment:
Considering there may be lint errors or warnings, what about something like lint problems?
You can choose your word.